action.yml

```diff
@@ -9,3 +9,8 @@ branding:
 runs:
   using: docker
   image: Dockerfile
+
+inputs:
+  repo-token:
+    description: "You probably want it to be ${{ secrets.GITHUB_TOKEN }}"
```

Comment:
`${{ secrets.GITHUB_TOKEN }}` needs to be escaped so that github actions don't try to insert something here

Reply:
Do you have evidence of bad behavior? It's just a description field and since this is not a workflow file I'm not sure if it would replace variables.

Reply:
https://github.com/RebeccaStevens/deepmerge-ts/runs/4923153760?check_suite_focus=true

Reply:
Oh, I see. Thank you! #324